queue/src/fiskaltrust.Middleware.Localization.QueueDE/JournalProcessorDE.cs

```diff
@@ -151,7 +152,11 @@ private async IAsyncEnumerable<JournalResponse> ProcessTarExportFromTSEAsync(Jou
         {
             var exportSession = await _deSSCDProvider.Instance.StartExportSessionAsync(new StartExportSessionRequest()).ConfigureAwait(false);
             var sha256CheckSum = "";
-            using (var memoryStream = new MemoryStream())
+
+            byte[] chunk;
+            List<byte> chunkList;
```

Comment:
What do we need this `chunkList` for? Couldn't this just be inside the `else` clause?

Reply:
I.e. `response.Chunk = chunk.ToList()`?